easybuild/easyconfigs/t/tbb/tbb-2021.2.0_relaxed.patch

```diff
@@ -0,0 +1,48 @@
+--- include/oneapi/tbb/detail/_concurrent_queue_base.h	2021-01-27 21:03:49.000000000 +0100
```

Comment:
@robert-mijakovic please include a comment explaining the purpose of the patch

Reply:
To be honest I remember I had some issue compiling it and then I introduced this patch but don't remember the reason. I have just triggered the build but couldn't reproduce it. Maybe it was affecting the previous release of tbb.

Reply:
I think the issue you experienced while testing is the one I faced. I will reenable the patch,  put the comment in EC and ask you to retrigger your test.

Reply:
@robert-mijakovic still seeing this build error:
```
[ 76%] Building CXX object test/CMakeFiles/conformance_concurrent_vector.dir/conformance/conformance_concurrent_vector.cpp.o
cd /mnt/ds918plus/eb/nuc-fedora32/build/tbb/2021.2.0/GCCcore-10.3.0/easybuild_obj/test && /mnt/ds918plus/eb/nuc-fedora32/software/GCCcore/10.3.0/bin/g++  -I/mnt/ds918plus/eb/nuc-fedora32/build/tbb/2021.2.0/GCCcore-10.3.0/oneTBB-2021.2.0/test/.. -I/mnt/ds918plus/eb/nuc-f
edora32/build/tbb/2021.2.0/GCCcore-10.3.0/oneTBB-2021.2.0/test -I/mnt/ds918plus/eb/nuc-fedora32/build/tbb/2021.2.0/GCCcore-10.3.0/oneTBB-2021.2.0/src/tbb/../../include -O2 -ftree-vectorize -march=native -fno-math-errno -O3 -DNDEBUG -Wall -Wextra -Werror -Wfatal-errors -
Wshadow -Wcast-qual -Woverloaded-virtual -Wnon-virtual-dtor -mrtm -pthread -std=c++11 -MD -MT test/CMakeFiles/conformance_concurrent_vector.dir/conformance/conformance_concurrent_vector.cpp.o -MF CMakeFiles/conformance_concurrent_vector.dir/conformance/conformance_concu
rrent_vector.cpp.o.d -o CMakeFiles/conformance_concurrent_vector.dir/conformance/conformance_concurrent_vector.cpp.o -c /mnt/ds918plus/eb/nuc-fedora32/build/tbb/2021.2.0/GCCcore-10.3.0/oneTBB-2021.2.0/test/conformance/conformance_concurrent_vector.cpp
In file included from /mnt/ds918plus/eb/nuc-fedora32/build/tbb/2021.2.0/GCCcore-10.3.0/oneTBB-2021.2.0/src/tbb/../../include/oneapi/tbb/concurrent_queue.h:21,
                 from /mnt/ds918plus/eb/nuc-fedora32/build/tbb/2021.2.0/GCCcore-10.3.0/oneTBB-2021.2.0/test/conformance/conformance_concurrent_queue.cpp:24:
/mnt/ds918plus/eb/nuc-fedora32/build/tbb/2021.2.0/GCCcore-10.3.0/oneTBB-2021.2.0/src/tbb/../../include/oneapi/tbb/detail/_concurrent_queue_base.h: In function void TestConstructors() [with CQ = ConcQWithSizeWrapper<Bar>; T = Bar; TIter = BarIterator; CQ_EX = ConcQWithSi
zeWrapper<BarEx>; T_EX = BarEx]:
/mnt/ds918plus/eb/nuc-fedora32/build/tbb/2021.2.0/GCCcore-10.3.0/oneTBB-2021.2.0/src/tbb/../../include/oneapi/tbb/detail/_concurrent_queue_base.h:547:29: error: dqb_ex.tbb::detail::d1::concurrent_queue_iterator_base<BarEx, tbb::detail::d1::cache_aligned_allocator<BarEx>
 >::my_array[7] may be used uninitialized in this function [-Werror=maybe-uninitialized]
  547 |                 my_array[i] = other.my_array[i];
      |                 ~~~~~~~~~~~~^~~~~~~~~~~~~~~~~
compilation terminated due to -Wfatal-errors.
cc1plus: all warnings being treated as errors
make[2]: *** [test/CMakeFiles/conformance_concurrent_queue.dir/build.make:79: test/CMakeFiles/conformance_concurrent_queue.dir/conformance/conformance_concurrent_queue.cpp.o] Error 1
make[2]: Leaving directory '/mnt/ds918plus/eb/nuc-fedora32/build/tbb/2021.2.0/GCCcore-10.3.0/easybuild_obj'
make[1]: *** [CMakeFiles/Makefile2:3085: test/CMakeFiles/conformance_concurrent_queue.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
```

Reply:
I have just reenabled the patch. Sorry for the slowness.

Reply:
@verdurin May I ask you how do you create a test report?

Reply:
@robert-mijakovic if you have the Github integration already configured, it's pretty simple. See https://docs.easybuild.io/en/latest/Integration_with_GitHub.html#uploading-test-reports-upload-test-report